assets/src/dashboard/app/views/shared/bodyWrapper.js

```diff
@@ -22,6 +22,10 @@ import styled from 'styled-components';
 const BodyWrapper = styled.div`
   margin: ${({ theme }) => `0 ${theme.pageGutter.small.desktop}px`};
 
+  @media ${({ theme }) => theme.breakpoint.largeDisplayPhone} {
+    margin: ${({ theme }) => `0 ${theme.pageGutter.small.min}px`};
```

Comment:
For the sake of my grid update and needing breakpoints to match with values would you mind updating this to be: 
`0 ${theme.pageGutter.small.largeDisplayPhone}px`};` 

And then in the theme change `pageHorizontalGutter.largeDisplayPhone` to `10`. 
I will update the gutters to use consistent structure tomorrow or friday.